endoreg_db/migrations/0003_remove_examination_description_de_and_more.py

```
@@ -0,0 +1,93 @@
+            model_name='findingintervention',
+            name='description_en',
+        ),
+        migrations.RemoveField(
+            model_name='findingintervention',
+            name='name_de',
+        ),
+        migrations.RemoveField(
+            model_name='findingintervention',
+            name='name_en',
+        ),
+    ]
```

Ruff (0.14.8)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)